jenkins-backup.sh

```diff
@@ -83,6 +86,10 @@ function main() {
     cp -R "${JENKINS_HOME}/nodes/"* "${ARC_DIR}/nodes"
   fi
 
+  if [ "$(ls -A ${JENKINS_HOME}/labels/)" ] ; then 
+    cp -R "${JENKINS_HOME}/labels/"* "${ARC_DIR}/labels"
```

Comment:
**[MUST]** Add a test below to see if the labels directory is really archived

https://github.com/sue445/jenkins-backup-script/blob/85e45190cb4d180fbbc200a92851a3c0b34ec7d8/spec/jenkins-backup_spec.rb#L1-L27